WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -306,6 +306,11 @@
         <%= f.input :extra_registration_requirements, input_html: { class: disable_form ? "" : "markdown-editor" } %>
 
         <hr>
+        <% if @competition.uses_qualification? %>
```

Comment:
This is tricky because it uses a method on `competition` while filling out a form that modifies that same competition. I'd rather have the Yes/No dropdown be controlled by jQuery, much like `qualification_results_reason`.

(Perhaps you can even use the same jQuery `.on('change'` body for both? Cf. `assets/javascripts/competitions.js:30`

Reply:
Sounds good, done.